Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Core/Src/Training.c

```diff
@@ -1,6 +1,42 @@
 #include "Training.h"
 
 void trainingTask(uint8_t* data)
-{
+{	
+	validData = 0;
+	uint8_t motor1Data = data[0];
+	uint8_t motor2Data = data[1];
+	int forward1 = motor1Data >> 7;						// Select bit 7 of motor 1
+	int forward2 = motor2Data >> 7;						// Select bit 7 of motor 2
```

Comment:
The instructions say that the forward/reverse are at bit 0. By doing this, you're getting bit 7

Reply:
Just fixed that